connectors/byoc.py

```diff
@@ -84,11 +84,10 @@ class DataSourceError(Exception):
 class ConnectorIndex(ESIndex):
     def __init__(self, elastic_config):
         logger.debug(f"ConnectorIndex connecting to {elastic_config['host']}")
-        # initilize ESIndex instance
+        # initialize ESIndex instance
```

Comment:
While I think it's okay to keep it in this PR, ideally renames/typo fixes should go in a different PR, especially if there are many of them - they just distract from the actual change IMO.